src/main/java/atdd/path/web/FavoriteController.java

```diff
@@ -0,0 +1,73 @@
+package atdd.path.web;
+
+import atdd.path.application.FavoriteService;
+import atdd.path.application.dto.FavoritePathResponseView;
+import atdd.path.application.dto.FavoriteStationResponseView;
+import atdd.path.domain.User;
+import atdd.path.security.LoginUser;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+import static atdd.path.application.base.BaseUriConstants.*;
+
+@RestController
+@RequestMapping(FAVORITE_BASE_URL)
+public class FavoriteController {
+
+    private FavoriteService favoriteService;
+
+    public FavoriteController(FavoriteService favoriteService) {
+        this.favoriteService = favoriteService;
+    }
+
+    @PostMapping(value = STATION_BASE_URL + "/{id}")
+    public ResponseEntity<FavoriteStationResponseView> createFavoriteStation(@PathVariable(name = "id") Long stationId,
+                                                                             @LoginUser User user) {
+        FavoriteStationResponseView favoriteStation = favoriteService.saveFavoriteStation(stationId, user);
+        return ResponseEntity.created(URI.create(FAVORITE_BASE_URL + STATION_BASE_URL + "/" + favoriteStation.getId()))
+                .body(favoriteStation);
+    }
+
+    @GetMapping(value = STATION_BASE_URL + "/{id}")
+    public ResponseEntity<FavoriteStationResponseView> retrieveFavoriteStation(@PathVariable Long id,
+                                                                               @LoginUser User user) {
+        FavoriteStationResponseView favoriteStation = favoriteService.findFavoriteStation(id, user);
+        return ResponseEntity.ok().body(favoriteStation);
+    }
+
+    @GetMapping
```

Comment:
컨벤션 대로라면 아래와 같이 되어야 할 것 같아요 :)
```suggestion
    @GetMapping(value = STATION_BASE_URL)
```